src/web/mjs/connectors/OneTwoThreeHon.mjs

```diff
@@ -0,0 +1,42 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+export default class OneTwoThreeHon extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'onetwothreehon';
+        super.label = '123hon';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://www.123hon.com';
+        this.mangaList = '/polca/web-comic/';
+
+        this.queryMangas = 'ul.comic__list > li > a';
+        this.queryChapters = 'div.read-episode li';
+    }
+
+    async _getMangas() {
+        const request = new Request(new URL(this.mangaList, this.url), this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryMangas);
+        return data.map(link => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(link, request.url),
```

Comment:
Consider `this.url` instead of `request.url` since the `URL` created in `_getChapters()` is also using `this.url` to rebuild an `uri` from the `manga.id`

Reply:
Missed that one.